tests/bluetooth/bsim/host/adv/encrypted/src/peripheral.c

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+#define DATA_SIZE 6
+static uint8_t data[] = {0x5A, 0x65, 0x70, 0x68, 0x79, 0x72};
+BUILD_ASSERT(sizeof(data) == DATA_SIZE);
+
+static uint8_t session_key[] = {0xC0, 0xC1, 0xC2, 0xC3, 0xC4, 0xC5, 0xC6, 0xC7,
+				0xC8, 0xC9, 0xCA, 0xCB, 0xCC, 0xCD, 0xCE, 0xCF};
```

Comment:
If the `session_key` needs to be identical between the central and peripheral, I suggest to put it (and any other common values) in a common file.